utilities/pipelines/staticValidation/module.tests.ps1

```diff
@@ -52,13 +61,13 @@ Describe 'File/folder tests' -Tag 'Modules' {
                 ($hasARM -or $hasBicep) | Should -Be $true
         }
 
-        It '[<moduleFolderName>] Module should contain a [` readme.md `] file.' -TestCases $moduleFolderTestCases {
+        It '[<moduleFolderName>] Module should contain a [` README.md `] file.' -TestCases $moduleFolderTestCases {
 
             param(
                 [string] $moduleFolderPath
             )
 
-            $pathExisting = Test-Path (Join-Path -Path $moduleFolderPath 'readme.md')
+            $pathExisting = Test-Path (Join-Path -Path $moduleFolderPath 'README.md')
```

Comment:
```suggestion
            $readMeFilePath = Join-Path -Path $moduleFolderPath 'README.md'
            $pathExisting = Test-Path $readMeFilePath
```

Reply:
@eriqua, this goes together with the suggestion below :) I wasn't abel to do it in one